.custom-format.py

```diff
@@ -29,14 +29,14 @@
 if CLANG_FORMAT_BIN is None:
     o = 0
     try:
-        p = subprocess.Popen(["clang-format-8", "--version"], stdout=subprocess.PIPE)
+        p = subprocess.Popen(["clang-format-10", "--version"], stdout=subprocess.PIPE)
```

Comment:
On arch, I have clang-format version 10.0.0 installed but `clang-format-10` is not in my path. 
I need to use `CLANG_FORMAT_BIN=clang-format make code-format` but that feels off.

Reply:
why dont you put a symlink in /usr/local/bin?

Reply:
I'm not a fan of messing with my distro's PATH, and there is no actual benefit (if an upgrade upgrades clang-format, the symlink won't change)